hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/exception/SCMSecurityException.java

```diff
@@ -109,6 +109,7 @@ public ErrorCode getErrorCode() {
     MISSING_BLOCK_TOKEN,
     BLOCK_TOKEN_VERIFICATION_FAILED,
     GET_ROOT_CA_CERT_FAILED,
-    NOT_A_PRIMARY_SCM
+    NOT_A_PRIMARY_SCM,
+    GET_RECON_CERTIFICATE_FAILED
```

Comment:
I think we can reuse `GET_CERTIFICATE_FAILED` instead of adding new error code.

Reply:
Shouldn't Recon as a separate component in the environment have its own Certificate Failure code, instead of using a generic one?

Reply:
I don't think we handle any of these component-specific codes differently (they are all part of the same `if` condition).